mail_template_conditional_attachment/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Druidoo - Iván Todorovich <ivan.todorovich@druidoo.io>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Email Template Conditional Attachment",
+    "summary": "Allow to add conditional attachments to email templates",
+    "version": "12.0.1.0.0",
+    "category": "Uncategorized",
+    "website": "https://github.com/OCA/social",
+    "author": "Druidoo, Odoo Community Association (OCA)",
+    "maintainers": [
+        "ivantodorovich"
+    ],
+    "license": "AGPL-3",
+    "depends": [
+        "mail",
+    ],
+    "data": [
+        "views/mail_template.xml",
+        "security/ir.model.access.csv",
```

Comment:
I think that is a best practice place the security files before all XML files. Can you load first "security/ir.model.access.csv" ?

Reply:
Thanks for the review! 
Done here https://github.com/OCA/social/pull/521/commits/bb97067a14ea2708862b9ce9292f7fb2bb557cb1 :+1: